package.json

```diff
@@ -58,18 +60,24 @@
     "i18next-json-sync": "^2.2.0",
     "jest": "^22.4.3",
     "jest-glamor-react": "^4.2.1",
+    "node-sass": "^4.8.3",
+    "patternfly-react": "^1.19.1",
     "react": "^16.3.1",
     "react-dom": "^16.3.1",
     "react-hot-loader": "^4.0.1",
     "react-i18next": "^7.5.1",
     "react-router-dom": "^4.2.2",
+    "react-svg-loader": "^2.1.0",
```

Comment:
This does not look to be used yet. Let's remove it for now, and we can add it with its use.